tests/helpers/util.ts

```diff
@@ -1,6 +1,7 @@
 import { TSESLint } from "@typescript-eslint/experimental-utils";
 import deepMerge, { Options as deepMergeOptions } from "deepmerge";
 import { Linter, Rule, RuleTester as ESLintRuleTester } from "eslint";
+import { filename } from "./configs";
```

Comment:
This should probably be in its own import block.
i.e. add a new line above this line to separate library imports from local imports.

Reply:
I usually just sort the imports with package imports at top and local imports at bottom without any separator. If we want to use a separator it would be nice if it could be enforced. Perhaps there is something in `eslint-plugin-import` we can use.